src/app/[lng]/(exchange)/exchange/page.tsx

```diff
@@ -1,9 +1,25 @@
-import dynamic from 'next/dynamic';
+import { cookies } from 'next/headers';
+import App from 'src/app/ui/app/App';
+import type { ThemeModesSupported } from 'src/types/settings';
 
-const App = dynamic(() => import('../../../ui/app/App'), { ssr: false });
+export const dynamic = 'force-dynamic';
 
 const Page = () => {
-  return <App starterVariant="default" />;
+  const activeTheme = cookies().get('theme')?.value as
+    | ThemeModesSupported
+    | undefined;
+  const welcomeScreenClosed = cookies().get('welcomeScreenClosed')?.value as
+    | 'false'
+    | 'true'
+    | undefined;
```

Comment:
We might want to design a dedicated hook for this in the future.

Reply:
Nice idea! Done